test/test_w3c_spec/test_sparql10_w3c.py

```diff
@@ -0,0 +1,128 @@
+"""
+Runs the SPARQL 1.0 test suite from.
```

Comment:
@aucampia - whoops, looks like a citation was truncated.

Reply:
AFAICT, the relevant (resolvable) URL is `https://www.w3.org/2001/sw/DataAccess/tests/data-r2`